.github/workflows/test.yml

```
@@ -0,0 +1,42 @@
+name: 'bugsnag-dsym-upload tests'
+
+on: push
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: ['2.5']
+        os: [ubuntu-20.04, macos-10.15]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+
+    - name: install dependencies (Ubuntu only)
+      if: contains(matrix.os, 'ubuntu')
+      run: |
+        sudo apt-get -qq update
+        sudo apt-get install -y llvm
+        sudo ln -s $(which llvm-dwarfdump) $(dirname $(which llvm-dwarfdump))/dwarfdump
+
+    - run: mkdir -p vendor/{fastlane-bundle,bundle}
+
+    - name: run specs
+      run: |
+        bundle config path vendor/fastlane-bundle
+        bundle install --gemfile=tools/fastlane-plugin/Gemfile
+        cd tools/fastlane-plugin
+        bundle exec rake spec
```

```suggestion
        bundle exec rspec
```

as we've removed the rake `spec` task in one of the PRs that'll merge into this.

---

That's already handled in the next PR: https://github.com/bugsnag/bugsnag-dsym-upload/pull/56/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88